app/views/follow_requests/_form.html.erb

```diff
@@ -1,19 +1,19 @@
-<%= form_with(model: follow_request, class: "d-inline-block") do |form| %>
-  <% if follow_request.errors.any? %>
-    <div id="error_explanation">
-      <h2><%= pluralize(follow_request.errors.count, "error") %> prohibited this follow_request from being saved:</h2>
+  <%= form_with(model: follow_request, local: false, id: "follow_request_form", class: "d-inline-block") do |form| %>
```

Comment:
Great job using local:false! The "id:'follow_request_form'" is not necessary here because you aren't calling it within your js.erbs